Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -22,6 +24,8 @@
 
 VENDOR_NAME = 'AutoFocus V2'
 
+DOMAIN_RE = re.compile(r'^[a-zA-Z\d-]{,63}(\.[a-zA-Z\d-]{,63})*$')
```

Comment:
we have that in commonserverpython

Reply:
I can't find it there - are you sure?

Reply:
My bad, we should add it there then @reutshal what is the most updated regex that we have for domain?

Reply:
We don't actually have a regex for domains because it is too general and can match to non-domain texts (every text in the form  _._ )
in the indicator extraction, we use url or email regex and then extracting the domain from the result.

Reply:
My guess is that we need to do something like that here too

Reply:
@roysagi ?